include/functions.h

```diff
@@ -2707,7 +2707,7 @@ UNK_TYPE func_8013BC6C(SkelAnime* skelAnime, ActorAnimationEntryS* arg1, s32 arg
 // void func_8013C068(UNK_TYPE1 param_1, UNK_TYPE1 param_2, UNK_TYPE1 param_3, UNK_TYPE1 param_4, UNK_TYPE4 param_5);
 // void func_8013C624(UNK_TYPE1 param_1, UNK_TYPE1 param_2, UNK_TYPE1 param_3, UNK_TYPE1 param_4, UNK_TYPE4 param_5, UNK_TYPE4 param_6);
 // void func_8013C8B8(void);
-void func_8013C964(Actor* actor, GlobalContext* globalCtx, f32 arg2, f32 arg3, s32 arg4, s32 arg5);
+void func_8013C964(Actor* actor, GlobalContext* globalCtx, f32 param_3, f32 param_4, s32 param_5, s16 param_6);
```

Comment:
This is fine as is if you want as all these args will be temporary, but I've been trying to change the args the other way. Anytime I see `params` in a func I'm using, I've been switching them to `arg` instead of params to make it more consistent with OoT.

Reply:
I prefer arg as well, but I'm personally not picky on undocumented code. Also there's no need for consistency to oot in undocumented code.